client/Cargo.toml

```diff
@@ -1,19 +1,19 @@
 [package]
 name = "localtunnel-client"
-version = "0.1.1"
-edition = "2021"
+version.workspace = true
+edition.workspace = true
 description = "A client to connect with localtunnel server."
-license = "MIT"
-repository = "https://github.com/kaichaosun/rlt"
+license.workspace = true
+repository.workspace = true
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-reqwest = { version = "0.11", features = ["json"], default-features = false }
-serde = { version = "1.0", features = ["derive"] }
-tokio = { version = "1.23.0", features = ["full"] }
-anyhow = { version = "1.0", features = ["backtrace"] }
-log = "0.4"
+reqwest = { version = "0.11", default-features = false }
```

Comment:
We still need the `json` feature here since it's used in client.
If the `reqwest` is not reused by crates, can you remove it in `workspace.dependencies`?